GeneratorInterface/GenFilters/interface/MCFilterZboostHelper.h

```diff
@@ -0,0 +1,16 @@
+#ifndef MCFilterZboostHelper_h
```

Comment:
hi @zc11 - actually I realize CMS has a few implementations of this already. Please use one or adopt the others to use this one - which may require some small refactoring . (https://cmssdt.cern.ch/lxr/search?_filestring=&_string=HepMC%3A%3AFourVector+zboost)

Reply:
Thanks for the review.
I've modified the mentioned 3 filters to use MCfilterZboostHelper
Please let me know if it's ok